data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -217,28 +219,8 @@ def _create_score_data(
             right_on=self.STATE_CODE_COLUMN,
             how="left",
         )
-
```

Comment:
Now that we truncate the tiles in the main score to the 2010 tract list, we don't need to do any of this here, which saves us several steps.

Reply:
this is lovely